### ISSUE TITLE
Story content picker and knobs covered by keyboard on mobile view

### ISSUE THREAD

**Issue by setyongr** [ookami-kb/storybook_flutter#129]
Hi, i've got an issue when running storybook in mobile device.

When i open the story picker or knobs, then tap the text field, the keyboard will cover the dialog.

Also, i think the dialog is too small on mobile device.

<img width="357" alt="image" src="https://github.com/ookami-kb/storybook_flutter/assets/10917018/6fcb11cc-3eb8-442e-8d10-2d76d5d7fb62">

**Comment by ookami-kb** [comment]
Yeah, good point, I'll take a look.

**Comment by bkkoluch** [comment]
Hey there @ookami-kb, I'd be grateful if you could take a look, it is a bit restricting for testers and creating a fork seems unnecessary 😄

**Comment by ookami-kb** [comment]
Yes, I'm currently working on a new layout that should solve this and some other issues.
